src/core/server/http/http_server.ts

```diff
@@ -69,10 +69,12 @@ export interface HttpServerSetup {
     get: AuthStateStorage['get'];
     isAuthenticated: AuthStateStorage['isAuthenticated'];
   };
+  createNewServer?: (config: Partial<HttpConfig>) => Promise<HttpServerSetup>;
```

Comment:
Should this be here? Seems like this is covered by the HttpServiceSetup interface